tests/typed_ir.tests

```diff
@@ -41,3 +41,8 @@ Bound quantifiers in a query (#694)
 tests/typed_ir/T694.links
 filemode : true
 stdout : fun : forall a::Row,b::Row(Any,Base).() -a-> [(|b::(Any,Base))]
+
+Curry (#574)
+tests/typed_ir/T574.links
+filemode : true
+stdout : () : ()
```

Comment:
Notice how github complains about lack of newline at the EOF. Running `git diff` before committing should show you a warning about this:

![1](https://user-images.githubusercontent.com/1597921/79217399-ba804380-7e46-11ea-9c27-ac52b8a22c00.jpg)

Reply:
That's how it's supposed to be. The test suite will complain if the `.tests` file ends with a newline.

Reply:
It will complain if you add **two** newlines at the end of file, but not when you add **one** newline. Note that `git diff` will also mark empty lines at the end of file, again making it easy to spot:

![3](https://user-images.githubusercontent.com/1597921/79219770-c837c800-7e4a-11ea-8557-4324dc3d82d4.jpg)

Reply:
You're right, sorry. I fixed this now.